primitives/src/prf.rs

```diff
@@ -7,9 +7,11 @@
 //! This module implements a pseudo random function that is derived from
 //! the rescue hash function.
 
+use core::marker::PhantomData;
```

Comment:
curious, should we use from `core` or `ark_std`, (I think the latter just re-export the `core` correct?) I wonder if there would be version incompatibility issue if we don't consistently using those reexported from `ark_std`

Reply:
Good catch. Indeed `ark-std` is just re-exporting `core::marker`. Let me replace this usage to ark_std, and make a separate PR that replaces all remaining usages (since the repo rn is inconsistent).